textattack/transformations/transformation.py

```diff
@@ -18,11 +18,17 @@ def __call__(self, current_text, pre_transformation_constraints=[], indices_to_m
                 beginning the transformation.
             indices_to_modify: Which word indices should be modified as dictated by the
                 ``SearchMethod``.
+            shifted_idxs (bool): Whether indices have been shifted from
```

Comment:
This is kinda unclear. I think it should be named `original_idxs` and say something like "Whether indices represent those from the original text rather than the current one."